packages/expo-av/src/AV.ts

```diff
@@ -272,8 +283,16 @@ export const PlaybackMixin = {
     });
   },
 
-  async setRateAsync(rate: number, shouldCorrectPitch: boolean): Promise<PlaybackStatus> {
-    return ((this as any) as AV).setStatusAsync({ rate, shouldCorrectPitch });
+  async setRateAsync(
+    rate: number,
+    shouldCorrectPitch: boolean,
```

Comment:
I guess it should be `false` by default 🤔 
```suggestion
    shouldCorrectPitch: boolean = false,
```